spec/requests/payments_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
```

Comment:
- [ ] Kindly make sure that all tests are passed, as there are 4 examples not passed.
![image](https://user-images.githubusercontent.com/24703171/230432099-a917d361-bd62-4937-93c6-a1afa044cec2.png)
- [ ] Good job on implementing tests for the models but kindly also include unit and integration tests for components(Sign up, log in, Home, Category, and Transaction pages) as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=Create%20unit%20and%20integration%20tests%20for%20all%20the%20most%20important%20components%20of%20your%20RoR%20application.).